resource_customizations/apps/StatefulSet/actions/action_test.yaml

```
@@ -2,3 +2,8 @@ actionTests:
 - action: restart
   inputPath: testdata/statefulset.yaml
   expectedOutputPath: testdata/statefulset-restarted.yaml
+- action: scale
+  inputPath: testdata/statefulset.yaml
+  expectedOutputPath: testdata/statefulset-scaled.yaml
+  parameters:
+    scale: "6"
```

```suggestion
    scale: "6"
```

(Don't trigger missing newline at eof)

---

I see no newline at eof